API/Services/Tasks/Scanner/ProcessSeries.cs

```diff
@@ -880,6 +880,8 @@ private void AddOrUpdateFileForChapter(Chapter chapter, ParserInfo info, bool fo
             existingFile.FileName = Parser.Parser.RemoveExtensionIfSupported(existingFile.FilePath);
             existingFile.FilePath = Parser.Parser.NormalizePath(existingFile.FilePath);
             existingFile.Bytes = fileInfo.Length;
+            existingFile.KoreaderHash = KoreaderHelper.HashContents(existingFile.FilePath);
```

Comment:
FYI, I added this as your original PR did not add the Hash for existing files. I added a safeguard on the KoreaderHelper to exit early for non-epub files.